cypress/support/dictionary/permissions.js

```diff
@@ -1011,6 +1011,10 @@ export default {
     internal: 'ui-users.settings.departments.create.edit.view',
     gui: 'Settings (Users): Can create, edit, and view department',
   },
+  consortiaSettingsConsortiumManagerPatronGroupsAll: {
+    internal: 'ui-users.settings.usergroups.all',
+    gui: 'Settings (Users): Can create, edit and remove patron groups',
```

Comment:
permission already exists: uiUsersCreatePatronGroups

Reply:
the one you've mentioned doesn't contain .**ALL** permissions
![image](https://github.com/folio-org/stripes-testing/assets/18191661/5fc3d623-7807-4aa4-b4fc-dbd296ff6a4d)
![image](https://github.com/folio-org/stripes-testing/assets/18191661/51baade0-a76e-429d-a463-d884d4df4176)